src/rpc/misc.cpp

```diff
@@ -106,6 +110,9 @@ UniValue getinfo(const UniValue &params, bool fHelp)
     obj.pushKV("blocks", (int)chainActive.Height());
     obj.pushKV("timeoffset", GetTimeOffset());
     obj.pushKV("connections", nNodes);
+    obj.pushKV("peers_graph", (int)thinrelay.GetGraphenePeers());
+    obj.pushKV("peers_xthin", (int)thinrelay.GetThinBlockPeers());
+    obj.pushKV("peers_cmpct", (int)thinrelay.GetCompactBlockPeers());
```

Comment:
let's spell out graphene and compact.  There's no reason to save a few bytes and this will help the call be self-documenting.  "graph" is especially confusing.  My initial reaction was that this was somehow data the would let you construct a 2-d diagram (graph) of some property of all peers.  

Also, "timeoffset" and "testnet" don't use underscore.  We should be consistent (although personally I'd prefer either underscore or bumpy case) because inconsistency makes it harder for developers to work with an RPC.

Reply:
ok i'll update it but i'd like to leave the underscores, it's pretty ugly without them.  Also there are numerous instances of underscore being used throughout rpc...even just a few lines down there is unlocked_until and there are others such as hash_serialized, disk_size etc...